link/src/test/java/com/stripe/android/link/ui/wallet/WalletScreenTest.kt

```diff
@@ -47,6 +47,8 @@ internal class WalletScreenTest {
         onWalletCollapsedChevron().assertIsDisplayed()
         onWalletCollapsedPaymentDetails().assertIsDisplayed()
         onCollapsedWalletRow().assertIsDisplayed().assertHasClickAction()
+        onWalletPayButton().assertIsDisplayed()
```

Comment:
Can you also add tests for:
- the buttons are displayed when PMs are expanded
- the buttons are enabled/disabled correctly